pkg/apis/cnsoperator/config/cns.vmware.com_cnsvolumemetadata.yaml

```diff
@@ -0,0 +1,135 @@
+                  this volume is referenced.
+                type: string
+              labels:
+                additionalProperties:
+                  type: string
+                description: Labels indicates user labels assigned to the entity in
+                  the guest cluster. Should only be populated if EntityType is PERSISTENT_VOLUME
+                  OR PERSISTENT_VOLUME_CLAIM. CNS Operator will return a failure to
+                  the client if labels are set for objects whose EntityType is POD.
+                type: object
+              namespace:
+                description: Namespace indicates namespace of entity in guest cluster.
+                  Should only be populated if EntityType is PERSISTENT_VOLUME_CLAIM
+                  or POD. CNS Operator will return a failure to the client if namespace
+                  is set for objects whose EntityType is PERSISTENT_VOLUME.
+                type: string
+              volumenames:
+                description: VolumeNames indicates the unique ID of the volume. For
+                  volumes created in a guest cluster, this will be “<guestcluster-ID>-<UID>”
+                  where UID is the pvc.UUID in supervisor cluster. Instances of POD
+                  entity type can have multiple volume names. Instances of PV and
+                  PVC entity types will have only one volume name.
+                items:
+                  type: string
+                type: array
+            required:
+            - entityname
+            - entityreferences
+            - entitytype
+            - guestclusterid
+            - volumenames
+            type: object
+          status:
+            description: CnsVolumeMetadataStatus defines the observed state of CnsVolumeMetadata
+            properties:
+              volumestatus:
+                description: The last error encountered, per volume, during update
+                  operation. This field must only be set by the entity completing
+                  the update operation, i.e. the CNS Operator. This string may be
+                  logged, so it should not contain sensitive information.
+                items:
+                  description: CnsVolumeMetadataVolumeStatus defines the status of
+                    the last update operation on CNS for the given volume. Error message
+                    will be empty if no error was encountered.
+                  properties:
+                    errormessage:
+                      type: string
+                    updated:
+                      type: boolean
+                    volumename:
+                      type: string
+                  required:
+                  - updated
+                  - volumename
+                  type: object
+                type: array
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
```

Comment:
With status added as subResource, it is preventing operations like modify/update/add on status field. Please remove the `subresources` section